dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/RequestParamUtils.java

```diff
@@ -195,4 +205,10 @@ public static QueryItem parseQueryItem( String item, Function<String, QueryItem>
         return queryItem;
     }
 
+    @FunctionalInterface
+    public interface CheckedFunction<T, R>
```

Comment:
@jbee can you please help us find a good home for this 😅 I think this could be helpful for others 😄

Reply:
in that case we should move it to another package too

Reply:
it should be in the `dhis-api` module. I'd pick the `org.hisp.dhis.util` package.